src/notiModel/notiModel.vala

```diff
@@ -226,6 +227,13 @@ namespace SwayNotificationCenter {
                             resident = hint_value.get_boolean ();
                         }
                         break;
+                    case "transient":
+                        if (hint_value.is_of_type (GLib.VariantType.BOOLEAN)) {
+                            transient = hint_value.get_boolean ();
+                        } else if (hint_value.is_of_type (GLib.VariantType.INT32)) {
+                            transient = hint_value.get_int32() == 1;
```

Comment:
About this part, just to give some explanations: I saw a lot of tutorials/answers on forums that are using a int32 for the transient parameter instead of a boolean, that's why I prefer covering both cases.